spotbugs/src/main/java/edu/umd/cs/findbugs/detect/FindNullDeref.java

```diff
@@ -1492,9 +1461,6 @@ public void foundGuaranteedNullDeref(@Nonnull Set<Location> assignedNullLocation
         SortedSet<SourceLineAnnotation> knownNullLocations = new TreeSet<>();
         for (Location loc : sourceLocations) {
             SourceLineAnnotation sourceLineAnnotation = SourceLineAnnotation.fromVisitedInstruction(classContext, method, loc);
-            if (sourceLineAnnotation == null) {
-                continue;
-            }
```

Comment:
Removing the null check, since `SourceLineAnnotation.fromVisitedInstruction()` won't return null.